integreat_cms/cms/views/bulk_action_views.py

```diff
@@ -102,6 +103,14 @@ def get_queryset(self) -> Any:
             queryset = queryset.prefetch_public_translations()
         return queryset
 
+    def get_extra_filters(self) -> Q:
+        """
+        Get extra filters for get_queryset(). Overwrite this method to specify needed filters
+
+        :return: Q Object to be used as filter argument
+        """
+        return Q(region=self.request.region)
```

Comment:
Could you add a comment on why this is necessary?

Tbh I am at a bit of a loss here. I tried it out and indeed, when directly applying `.filter(Q(...))` it won't work. Why? 😕 😂

Reply:
`get_extra_filters` is needed to be able to overwrite the filter for `region` because `contact` gets its region from `location` due to the relationship between those two entities. So in `contact_bulk_actions.py` the method is overwritten to filter for `location__region`. If you apply `region=self.request.region` directly, the contact bulk actions won't work because `region` does not exist in the `contact`model. Returning a Q object is needed to be able to use it as argument for `.filter()`.

Reply:
Ohhhhh OK. That makes sense. THakns for the explanmation :D
It's not really intuitive though just from the code (at least IMO).

I think it would be helpful to either explain that `get_extra_filters` gets overriden by some views in its docstring, or to overwrite `get_queryset` instead in the `ContactBulkAction`.

@david-venhoff also feel free to weigh in on this if I am telling nonsense here ^^

Reply:
I suggested to create this method because overwriting `get_queryset` (as was done originally in this pr) was a bit hard to understand imo and lead to a bit of code duplication, but more documentation can't hurt :D

Reply:
I added a reference to `contact_bulk_action` in the docstring and some explanation in the docstring in the override in `contact_bulk_action`. Hope that it makes it better to understand.

Reply:
Thanks for your suggestions :)